Jenkinsfile

```diff
@@ -12,7 +12,7 @@ pipeline {
     DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
     REGISTRY = 'docker.elastic.co'
     STAGING_IMAGE = "${env.REGISTRY}/observability-ci"
-    GO_VERSION = '1.14.2'
+    GO_VERSION = '1.15.5'
```

Comment:
Uhm... 1.15.6 (with a few critical fixes) was released right after you changed it to 1.15.5 :)